tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
+            # poplate the volume options
+            reg_ex = re.compile("^volume[0-9].options+")
```

Comment:
I think this doesn't match "volume12.options". Should it be something like ("^volume[0-9]+.options+")